front50-bom/front50-bom.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "java-platform"
+apply plugin: "maven-publish"
+
+// without this building the pom fails when using the Nebula publishing plugin
+configurations {
+  create("compileOnly")
+}
+
+javaPlatform {
+  allowDependencies()
+}
+
+
+if (Boolean.valueOf(enablePublishing)) {
+  publishing {
+    publications {
+      nebula(MavenPublication) {
+        from components.javaPlatform
+      }
+    }
+  }
+}
+
+dependencies {
+  api(platform("com.netflix.spinnaker.kork:kork-bom:$korkVersion"))
+  api(platform("com.netflix.spinnaker.fiat:fiat-api:$fiatVersion"))
```

Comment:
[same comment as the gate PR](https://github.com/spinnaker/gate/pull/829#discussion_r294920252)